src/types.jl

```diff
@@ -197,9 +201,11 @@ function get_extent(face::FTFont, char::Char)
 end
 
 function internal_get_extent(face::FTFont, char::Char)
-    err = FT_Load_Char(face, char, FT_LOAD_DEFAULT)
+    err = FT_Load_Char(face, char, FT_LOAD_NO_SCALE)
```

Comment:
this could use a comment maybe

Load chars without scaling. This leaves all glyph metrics that can be retrieved in font units, which can be normalized by dividing with the font's units_per_EM. This is more robust than relying on extents that are only valid with a specific pixelsize, because a font's pixelsize can be silently changed by third parties, such as Cairo. If that happens, all glyph metrics are incorrect. We avoid this by using the normalized space.